### ISSUE TITLE
Register indentation offenses for hash values that start on the next line

### ISSUE THREAD

**Issue by lovro-bikic** [rubocop/rubocop#14483]
## Is your feature request related to a problem? Please describe.

Currently, I don't believe there's a cop in RuboCop which will register offenses for:
```ruby
{
  foo:
value, # value that's dedented relative to hash key
  bar:
  value, # value that's aligned with hash key
  baz:
         value # value that's too indented relative to hash key
}
```

## Describe the solution you'd like

A new cop or an expansion of an existing cop to enforce consistent indentation for hash values that start on the next line. I think the two accepted styles should be:
```ruby
{
  foo:
    value, # indented
  bar:
  value # aligned
}
```
(Personally, I think `indented` should be the default.)

I opened this as an issue here instead of in the style guide repo because I'm not sure if this is a false negative for an existing cop, or if it qualifies as a new rule.

**Comment by lovro-bikic** [comment]
Bump, I think this is still relevant.
